AddonTerraformTemplate/main.tf

```diff
@@ -38,14 +38,14 @@ resource "azurerm_kubernetes_cluster" "k8s" {
 resource "azurerm_monitor_data_collection_endpoint" "dce" {
   name                          = "MSProm-${var.monitor_workspace_location}-${var.cluster_name}"
   resource_group_name           = azurerm_resource_group.rg.name
-  location                      = azurerm_resource_group.rg.location
+  location                      = var.monitor_workspace_location
```

Comment:
thanks for the fixes grace. So is the issue - template using RG region, instead of MAC region , there by creating no data issues for clusters where RG and MAC are in 2 different regions ?

Reply:
Yes the template fails during deployment right now if the resource group and AMW region are different

Reply:
ok thanks Grace.